src/App.jsx

```diff
@@ -1,35 +1,78 @@
 import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
 import './App.css'
+import MovieCards  from './MovieCards'
+import Favorites  from './Favorites'
+import { Search } from './Search'
+
+import { create } from 'zustand'
+
+export const searchStore = create((set) =>
```

Comment:
it would be great if you can remove the console log and create a separate folder for the store and export the support from that folder, it will make your code more clean, scalable and manageable